apps/web/components/integrations/IntegrationListItem.tsx

```diff
@@ -14,7 +15,11 @@ function IntegrationListItem(props: {
   return (
     <ListItem expanded={!!props.children} className={classNames("flex-col")}>
       <div className={classNames("flex w-full flex-1 items-center space-x-2 p-3 rtl:space-x-reverse")}>
-        {props.imageSrc && <img className="h-10 w-10" src={props.imageSrc} alt={props.title} />}
+        {props.imageSrc && (
+          <div className="h-10 w-10">
+            <Image width={40} height={40} src={props.imageSrc} alt={props.title} layout="fixed" />
+          </div>
+        )}
```

Comment:
```suggestion
        {/* eslint-disable-next-line @next/next/no-img-element */}
        {props.imageSrc && <img className="h-10 w-10" src={props.imageSrc} alt={props.title} />}
```

Reply:
This will not work as it should be 
```
{/* eslint-disable-next-line @next/next/no-img-element */}{props.imageSrc && <img className="h-10 w-10" src={props.imageSrc} alt={props.title} />}
``` 
Right next to each other and prettier fixes that automatically unless you also disable prettier, but I found that in some cases prettier rule is ignored.

Reply:
Try then:
```js
{// eslint-disable-next-line @next/next/no-img-element
props.imageSrc && <img className="h-10 w-10" src={props.imageSrc} alt={props.title} />}
```

Reply:
Nice! Will do, thanks